### PR TITLE
feat(#1604) Add more informative message.

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
1. Added `defaultBranch` method to `Profile`
2. `defaultBranch` is added for log message
3. Test added 

I'm not sure that it's correct to add method to the `Profile` interface, but I didn't found any other way to get a branch.

Closes: #1604

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you have a look, please?

File: src/main/java/com/rultor/agents/req/StartsRequest.java [hunk 1]
```diff
@@ -348,8 +348,11 @@ private DockerRun docker(final String type) throws IOException {
         if (nodes.isEmpty()) {
             throw new Profile.ConfigException(
                 String.format(
-                    "There is no '%s' section in .rultor.yml",
-                    type
+                    "There is no '%s' section in %s for branch %s in repo %s",
+                    type,
+                    ".rultor.yml",
+                    "master",
```
Comment (yegor256) [inline src/main/java/com/rultor/agents/req/StartsRequest.java:354]: @volodya-lombrozo you meant `profile.defaultBranch()` here? typo?

Comment (volodya-lombrozo) [reply src/main/java/com/rultor/agents/req/StartsRequest.java]: @yegor256 Yes, forget about it. Fixed.

**Comment by volodya-lombrozo** [comment]
@yegor256 reminder

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**